src/cgame/cg_main.cpp

```diff
@@ -1119,7 +1119,6 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 	new(&cgs) cgs_t{}; // Using {} instead of () to work around MSVC bug
 	cg.~cg_t();
 	new(&cg) cg_t{};
-	memset( cg_entities, 0, sizeof( cg_entities ) );
```

Comment:
This is just deleting the initialization with no replacement. You could do `for(centity_t& ent : cg_entities) { ent = {}; }`